Ecosystem Partners/Marketplace Guides/getting-started-with-smspasscode-template.md

```diff
@@ -0,0 +1,128 @@
+
+### Description
+
+SMS PASSCODE's unique Adaptive Multifactor Authentication is among the most secure options available for adding additional protection over and above user name and password.  It generates a One Time Passcode in real time at the time of login and ties that code to the unique session initiated by the user.  It also supports Geo-location and Geo-fencing allow additional access control based on user location as well as greatly reduced attack surface.
+
+For more information, please visit https://www.smspasscode.com
+
+### Audience
+CenturyLink Cloud Users
+
+### Impact
+After reading this article, the user should feel comfortable getting started using SMS PASSCODE on CenturyLink Cloud.
+
+After executing the steps in this Getting Started document, the users will have a functioning SMS PASSCODE Authentication Server upon which they can start securing access for their Citrix, VPN-connections, Windows Login, Remote Desktop and some cloud services etc..
+
+### Prerequisite
+- Access to the CenturyLink Cloud platform as an authorized user.
+- Windows Server
+
+### Postrequisite
+
+To access your application from a computer outside the CenturyLink Cloud network, perform the following tasks after you receive the email notifying you that the Blueprint completed successfully:
+  1. [Add a Public IP](../Network/how-to-add-public-ip-to-virtual-machine.md) to your server through Control Portal
+  2. [Allow incoming traffic](../Network/how-to-add-public-ip-to-virtual-machine.md) for desired ports by clicking on the Servers Public IP through Control Portal and configuring appropriately.
+    * The default port to access the web admin interface is 2000
+
+### Deploying the SMSPasscode partner Template
+### How to Deploy the Partner Image
+  1. Create an email to ServiceTasks@ctl.io
+  2. Copy and paste the information below into the body of the email
+	3. Edit the information as needed and send
+
+   ```
+  TO: ServiceTasks@ctl.io
+
+  EMAIL SUBJECT:   Ecosystem Partner Template Import Request
+
+  CLC Support Team,
+
+  Please create a ticket to import the Ecosystem Partner Template image referenced below to my CenturyLink Cloud Account:
+
+  - Import CenturyLink Ecosystem Partner Source Image: SMS Passcode Base Server Template (from account SMSP)
+  - My CenturyLink Cloud Account Alias: ####
+  - My CenturyLink Cloud Account PIN:  ######
+  - Data Center to import image to: ###
+  - Server Name to import image as: ##########
+  - VLAN in the account to add the Server to: ########
+  - Additional Notes or work to be done: ########
+
+  Please let me know if you have any questions or issues. Kindly send me a reply once the work has been completed and let us know the IP address of the server where this technology has been deployed.
+
+  Thank you very much,
+
+	   Your_Name_Here
+	   ```
+Your account alias and PIN are available from your account info page and your user profile page respectively.
+

```

Comment:
your code text is not ending properly.  please review the preview as it requires fixing